mcs/build/profiles/monodroid.make

```diff
@@ -26,6 +26,7 @@ PROFILE_MCS_FLAGS = \
 	-d:MONODROID \
 	-d:ANDROID \
 	-d:NETSTANDARD \
+	-d:NO_CONSOLE \
```

Comment:
This also needs to be added to all profiles that defined `MOBILE` before so they continue to exclude console.